msal/build.gradle

```diff
@@ -125,9 +125,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.5', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.6', changing: true)
```

Comment:
the common version here   should be `0.0.7` for the change

Reply:
Thanks, resolved in d0bb6d1045c8c80f3aacb4b054ed45cf0c9cce0c